package.json

```diff
@@ -51,6 +51,7 @@
     "@types/node": "^10.3.4",
     "@types/react": "^16.4.0",
     "@types/storybook__react": "^3.0.7",
+    "autobind-decorator": "^2.1.0",
```

Comment:
How did we not have autobind before?

Reply:
This is in the root level so StoryBook can use it.